src/custom/pages/Faq/index.tsx

```diff
@@ -434,8 +437,12 @@ export default function Faq() {
             solutions.
           </p>
           <p>
-            In the future, the protocol might provide EIP712 support for off-chain signing also for smart contracts,
-            making gas-less trading possible also for smart contracts.
+            In the future, the protocol might provide{' '}
+            <ExternalLink href="https://eips.ethereum.org/EIPS/eip-1271" target="_blank" rel="noopener noreferrer">
+              EIP-1271
+            </ExternalLink>{' '}
```

Comment:
```suggestion
            </ExternalLink>{' '}
```

JSX is funny. I remember having an incredibly hard time having an inline `<tag>` followed by a line-break to not add whitespace in plain HTML (had to `<!-- comment -->` out the line-break), and it looks like in JSX you have the opposite issue :stuck_out_tongue:.